contents.json

```diff
@@ -5766,6 +5766,11 @@
       "category": "utility",
       "description": "📦 KeyPath dynamicMemberLookup based syntax sugar for Swift.",
       "homepage": "https://github.com/marty-suzuki/DuctTape"
+    }, {
+      "title": "BTree",
+      "category": "algorithm",
+      "description": "Fast sorted collections for Swift using in-memory B-trees",
```

Comment:
Hi @IceCreamCremo, could you add a full stop/period to the end of description?

Reply:
done